Source/objects.cpp

```diff
@@ -2772,7 +2772,7 @@ bool OperateShrineMagical(int pnum)
 	    player.position.tile,
 	    player._pdir,
 	    MIS_MANASHIELD,
-	    -1,
+	    TARGET_PLAYERS,

```

Comment:
This value will appear in save games, but it only has an effect for spawning and simply needs to be `!= TARGET_MONSTER`, so won't have an effect on comparability.